x/group/msgs.go

```diff
@@ -1,14 +1,14 @@
 package group
 
 import (
-	"github.com/cosmos/gogoproto/proto"
+	proto "github.com/cosmos/gogoproto/proto"
```

Comment:
Why?

Reply:
Same as above.  

I don't think I wrote that so I'm wondering how it got there. I know that at one point I checked out group from main.

But it is wierd.